manifests/integrations/kubernetes.pp

```diff
@@ -0,0 +1,45 @@
+# Class: datadog_agent::integrations::kubernetes
+#
+# This class will install the necessary configuration for the kubernetes integration
+#
+# Parameters:
+#   $url:
+#     The URL for kubernetes API
+#
+#   $tags:
+#     optional array of tags
+#
+#
+#
+# Sample Usage:
+#
+#   class { 'datadog_agent::integrations::kubernetes' :
+#     api_server_url  => 'https://kubernetes:443',
```

Comment:
You'll want to cover auth as well the the certificate options in kubernetes.yaml as well